src/base/private/profile_p.cpp

```diff
@@ -179,7 +181,7 @@ QString Private::Converter::toPortablePath(const QString &path) const
 
 QString Private::Converter::fromPortablePath(const QString &portablePath) const
 {
-    if (QDir::isAbsolutePath(portablePath))
+    if (portablePath.isEmpty() || QDir::isAbsolutePath(portablePath))
```

Comment:
Looks like auto torrent management isn't possible in portable mode without this additional check. Without it on every restart existing torrents are not in auto management mode anymore and moved automatically to `profile` folder.
I believe this is because of lines https://github.com/qbittorrent/qBittorrent/blob/e906478a16937236bdf583c3d36ef13175eaba20/src/base/bittorrent/session.cpp#L158-L159 and https://github.com/qbittorrent/qBittorrent/blob/e906478a16937236bdf583c3d36ef13175eaba20/src/base/bittorrent/torrenthandle.cpp#L203 Looks like in auto management mode `qBt-savePath` should be empty and this is causing wrong setting of `savePath` and then `m_useAutoTMM` in portable mode without this check.